From: Torsten Landschoff <t.landschoff@gmx.net>
Date: Tue, 30 Mar 1999 21:05:09 +0100
Subject: Use termios instead of termio (Closes: #35288).

Patch updated on Mon, 07 Mar 2011 20:40:53 +0100 based on
ispell-3.3.02-terminal.patch from ispell-3.3.02-102.1.src.rpm

For ispel 3.4.00 majority of the patch was removed, as the termios
support was integrated by upstream.
---
 term.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/term.c b/term.c
index 2ef8e7a..24bebcc 100644
--- a/term.c
+++ b/term.c
@@ -209,9 +209,13 @@ void terminit ()
     int			tpgrp;
 #else
 #ifdef TIOCGPGRP
+#ifdef USE_TERMIOS
+    pid_t			tpgrp;
+#else
     int			tpgrp;
 #endif
 #endif
+#endif
 #ifdef TIOCGWINSZ
     struct winsize	wsize;
 #endif /* TIOCGWINSZ */
